src/transforms/route.rs

```diff
@@ -38,15 +39,17 @@ impl SyncTransform for Route {
         event: Event,
         output: &mut vector_core::transform::TransformOutputsBuf,
     ) {
+        let mut passed: usize = 0;
```

Comment:
nit: The name `passed` initially confused me because I interpreted it as "condition check passed" not "passed by a condition without success". Maybe `check_failed` would be clearer?